examples/cookbook/src/lib.rs

```diff
@@ -186,4 +187,64 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    #[cfg(feature = "rocksdb")]
+    async fn use_created_db_rocksdb() -> Result<()> {
+        use fuels::accounts::fuel_crypto::SecretKey;
+        use fuels::accounts::wallet::WalletUnlocked;
+        use fuels::client::{PageDirection, PaginationRequest};
+        use fuels::prelude::{setup_test_provider, Config, DbType, ViewOnlyAccount};
+        use std::fs;
+        use std::path::PathBuf;
+        use std::str::FromStr;
+
+        // ANCHOR: use_created_rocksdb
+        let path =
```

Comment:
Maybe we should talk about this because I'm not really sure about this. I did some tests and I don't think this should be a problem. Of course, we can always delete example for loading an old db.

Reply:
Agreed with @segfault-magnet here. But I do think that proving that an old DB can be reloaded and "prove" that it keeps the state is good, just that it should be done in the same test.